fecgov/openFEC
# add state_full to candidates/totals/aggregates endpoint

## Summary (required)

- Resolves #5163 

This ticket adds state_full to the candidates/totals/aggregates endpoint. 

### Required reviewers 1-2 developers 


## Impacted areas of the application

General components of the application that this PR will affect:

-  candidates/totals/aggregates endpoint, aggregate by office-state and office-state-district 


## Related PRs

Related PRs against other branches: 
#5345 - database portion, needs to be merged in first

## How to test
1. Check out this branch
2. `pytest`
3. Test candidates/totals/aggregates endpoint on local 

Sample URLS: 
office-state sorted by state-full:
 
local:
http://127.0.0.1:5000/v1/candidates/totals/aggregates/?election_full=true&sort_hide_null=false&page=1&election_year=2022&aggregate_by=office-state&sort_nulls_last=false&sort_null_only=false&sort=state_full&per_page=20

compare to:
https://api.open.fec.gov/v1/candidates/totals/aggregates/?sort_nulls_last=false&election_year=2022&election_full=true&per_page=20&api_key=DEMO_KEY&sort_null_only=false&page=1&sort=-election_year&sort_hide_null=false&aggregate_by=office-state

office-state-district: 

local:
http://127.0.0.1:5000/v1/candidates/totals/aggregates/?election_full=true&sort_hide_null=false&min_election_cycle=2012&page=1&max_election_cycle=2022&aggregate_by=office-state-district&sort_nulls_last=false&sort_null_only=false&sort=-election_year&per_page=20

compare to: 
https://api.open.fec.gov/v1/candidates/totals/aggregates/?sort_nulls_last=false&election_full=true&max_election_cycle=2022&per_page=20&min_election_cycle=2012&api_key=DEMO_KEY&sort_null_only=false&page=1&sort=-election_year&sort_hide_null=false&aggregate_by=office-state-district
